packages/transaction-controller/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Fix: not used fixed gas limit for transactions of type 0x4 ([#5646](https://github.com/MetaMask/core/pull/5646))
```

Comment:
Should this go under `### Fixed`?

Maybe as:

`Do not use fixed gas for type 4 transactions`

Reply:
PR is updated